File: brute_force/Strings/StringPresentIncrementByCount.c
#include<stdio.h>
#include<string.h>

int main()
{
    int i,j;
    char str1[100],str2[100];
    int flag=0,c=0,flag1,k,k1;
    printf("Enter First String :");
    gets(str1);
    printf("Enter Second String :");
    gets(str2);
    int l1=strlen(str1);
    int l2=strlen(str2);
    
    for(i=0;i<l1;i++)
    {
    	if(str2[0]==str1[i])
    	{
    		flag1=0;
    		k=i;
    		k1=0;
    		for(j=0;j<l2;j++)
    		{
    			k=k+k1;
    			if(str2[j]!=str1[j+k])
    			{
    				flag1=1;
    				break;
    			}
    			else
    			{
    				k1++;
    			}
    		}
    		if(flag1==0)
    		{
    			printf("Match Found At Index %d\n",i);
    			c++;
    			flag=1;
    		}
    	}
    }
    
    printf("Count=%d",c);
      				
}

File: brute_force/Strings/StringPresentIncrementByCount.java
import java.util.*;

public class StringPresentIncrementByCount {


    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter Main String :");
        String str1=sc.nextLine();
        System.out.println("Enter Sub String :");
        String str2=sc.nextLine();
        
        int flag=0,flag1=1,count=0,k=0,k1=0;

        for(int i=0;i<str1.length();i++){
            if(str2.charAt(0)==str1.charAt(i));{
                flag1=0;
                k=i;
                k1=0;
                for(int j=0;j<str2.length();j++){
                    k=k+k1;
                    if(str2.charAt(j)!=str1.charAt(j+k)){
                        flag1=1;
                        break;
                    }
                    else{
                        k1++;
                    }
                }
                if(flag1==0){
                    System.out.println("String Matched at Index :"+i);
                    flag=1;
                    count++;
                }
            }
        }

        if(flag==1){
            System.out.println("Match Found :");
        }
        else{
            System.out.println("Match Not Found :");
        }
        System.out.println("String Occured By "+count+" Times");
    }
}